WORKSPACE

```diff
@@ -1048,7 +1048,7 @@ go_repository(
 
 # Change this and the docker/Dockerfile.proxy* files together
 # This SHA is obtained from proxy/postsubmit job
-ISTIO_PROXY_BUCKET = "1509b9361c3e727b40146dcddd44ed0012b7de78"
+ISTIO_PROXY_BUCKET = "a05c813aa01b8eed00d1698f5daba3f468b0a293"
```

Comment:
those changes are supposed to happen automatically with the bot (but maybe the bot is down?)

Reply:
yes. it is probably not operating yet after the mono-repo reshuffle.